exercises/1-the-manual-menace/README.md

```diff
@@ -308,25 +315,29 @@ ansible-playbook apply.yml -e target=tools \
 ![ocp-mongo](../images/exercise3/ocp-mongo.png)
 
 <p class="tip">
-Note - When making changes to the "enablement-ci-cd" repo, you should frequently commit and push the changes to git.
+<b>NOTE</b> - When making changes to the "enablement-ci-cd" repo, you should frequently commit the changes to git.
 </p>
 
 ### Part 5 - Jenkins & S2I
 > _Create a build and deployment config for Jenkins. Add new configuration and plugins to the OpenShift default Jenkins image using s2i_
 
 1. As before; create a new set of params by creating a `params/jenkins` file and adding some overrides to the template and updating the `<YOUR_NAME>` value accordingly.
-```
-MEMORY_LIMIT=3Gi
-VOLUME_CAPACITY=10Gi
-JVM_ARCH=x86_64
-NAMESPACE=<YOUR_NAME>-ci-cd
-JENKINS_OPTS=--sessionTimeout=720
+```yaml
+# params/jenkins
+
+  MEMORY_LIMIT=3Gi
```

Comment:
Looks like we gained some extra space indentations here ... probably shouldn't as copy/paste may be wrong.